PowerEditor/src/WinControls/WindowsDlg/WindowsDlg.cpp

```diff
@@ -865,6 +868,22 @@ void WindowsDlg::doClose()
 		}
 		ListView_SetItemCount(_hList, _idxMap.size());
 	}
+	doCount();

```

Comment:
Why do we need to count the document while closing the dialog?

Reply:
This is not closing the dialog, this is the button event which closes the chosen tab in the dialog, thus we need to update the count after we closing tabs